xcell/mappers/mapper_CatWISE.py

```diff
@@ -46,6 +46,32 @@ def get_signal_map(self, apply_galactic_correction=True):
             nmap_data = get_map_from_points(self.cat_data, self.nside,
                                             ra_name='ra',
                                             dec_name='dec')
+            
+            #### ecliptic latitude correction -- SvH 5/3/22
+            if apply_galactic_correction :
```

Comment:
OK, great. To make this more accurate, can you rename this variable `apply_ecliptic_projection`?
Also, according to PEP8, no space before `:`!

Reply:
Yep, deleted the space.